### DISCUSSION TITLE
Access denied for user 'root'@'ip-172-31-46-10.ec2.internal

### DISCUSSION THREAD

**Discussion by Mohandkajuluri** [learndevopsonline/learndevopsonline#1607] [Q&A]
Hi, 
I'm getting this error, could you please help with it.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/144593251/34d30759-6630-4b5f-bfcd-8e031c0e4974)

**Comment by sreedharm07** [comment]
Hi @Mohandkajuluri 
can you turn on all your machines and please give workstation ip

**Comment by Mohandkajuluri** [comment]
Hi, @sreedharm07 
workstation ip: 54.175.32.118
All machines are running.

**Reply by sreedharm07** [reply]
Hi @Mohandkajuluri 
you sql server was loaded with username **root** 
your parameter value was wrong please change it to **root** instead of **root/n**, it will works
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/ba184b89-673e-45db-92d9-8e64a8121b8e)

**Comment by Mohandkajuluri** [comment]
Hi, @sreedharm07 
Got it, it's working without errors.
Thank you so much.
